libbroker/broker/internal/store_actor.cc

```diff
@@ -66,11 +66,13 @@ store_actor_state::store_actor_state(caf::event_based_actor* selfptr)
   // nop
 }
 
-void store_actor_state::init(endpoint_id this_endpoint, endpoint::clock* clock,
+void store_actor_state::init(prometheus_registry_ptr reg,
+                             endpoint_id this_endpoint, endpoint::clock* clock,
                              std::string&& store_name, caf::actor&& core,
                              consumer_resource<command_message> in_res,
                              producer_resource<command_message> out_res) {
   BROKER_ASSERT(clock != nullptr);
+  this->registry = reg;
```

Comment:
Did you consider that the entire block is prefixed with `this->`? Having this one member assignment not use `this` would break the symmetry and the member variables would no longer align vertically, which I think improves the readability here.